src/Entities/ArrowEntity.cpp

```diff
@@ -116,7 +116,7 @@ void cArrowEntity::OnHitEntity(cEntity & a_EntityHit, Vector3d a_HitPos)
 	}
 
 	unsigned int PunchLevel = m_CreatorData.m_Enchantments.GetLevel(cEnchantments::enchPunch);
-	double KnockbackAmount = 11 + 10 * PunchLevel;
+	double KnockbackAmount = 10 + 8 * PunchLevel;
```

Comment:
I think it could make sense to gives these values names. It's hard to guess what these numbers mean.
`MinimumKnockbackAmount = 10;
PunchLevelMultiplier = 8` would make it IMHO clearer.